.github/workflows/check-and-test.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
         uses: DeterminateSystems/nix-installer-action@main
 
       - name: Test magic-nix-cache-action@main on ${{ matrix.systems.runner }}
-        uses: DeterminateSystems/magic-nix-cache-action@main
+        uses: DeterminateSystems/magic-nix-cache-action@graham/fh-433-magic-nix-cache-should-disable-github-actions-cache-if
```

Comment:
Not sure we want to have this  branch be used in production...? If it gets deleted, runs will start failing.

Reply:
EXTREME DERP